client/react-client-app/src/features/Menu.jsx

```
@@ -1,19 +1,46 @@
-import { hasUserSession } from "../utils/session";
+import { hasUserSession } from "../utils/session.js";
+import { Link } from "../components/Link.jsx";
 
 function Menu() {
     const hasSession = hasUserSession();
-    return <div role="navigation" className="vflex menu sidebar">
+
+    return <nav role="navigation" className="vflex menu sidebar">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<nav>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)